### DISCUSSION TITLE
Atom send a requests at startup to compute-X.amazon.com

### DISCUSSION THREAD

**Discussion by colorwebdesigner** [atom/atom#23528] [Q&A]
I noticed that every time I start the Atom editor it sends requests to `...compute-X.amazonaws.com` addresses. In general, this is not very good for a "hackable text editor", and in some situations even very bad. I look everything I could, but did not find a way to turn it off. 

I have only been able to find that the requests stop when running the editor with all custom packages removed. In my case it means that user folder `~/.atom/packages` is emty. 

As an experiment, I tried installing the `flex-tool-bar` package and restarting the editor. And again on startup I got 1 request to the Amazon server. The Flex toolbar has a dependency `tool-bar` and installs this package as well. After that, there are already two requests at startup. So, if you have a lot of custom modules - you have a lot of outgoing requests at startup.

I reviewed all the code of these modules, but did not find anything that could send these requests. Based on the results of my research, I came to the conclusion that the Atom package manager itself can send requests at startup, possibly checking for packages updates. But in my editor settings, the option "check for updates" is disabled. I have not found anywhere a description of a similar situation and a solution. 

If you understand the reason for this behavior and how to stop Atom to send a reqests, then please let me know.

I made a screencast about how it looks like: https://user-images.githubusercontent.com/18647764/151485852-c83d25e1-8ed7-4f27-b1e8-4be301181aa2.mp4

**Comment by ThatXliner** [comment]
try turning off telemetry

**Reply by colorwebdesigner** [reply]
The same idea was the first in my list ) Yes, It looks like telemetry, but i especially turn it off in settings. Everything that can send something somewhere was turned off.

![screen-28-01-2022_07-27](https://user-images.githubusercontent.com/18647764/151487376-e29b5535-b31b-4ae4-9a80-f60288040e97.png)

**Reply by colorwebdesigner** [reply]
You gave me another idea about the Version Pinned Packages setting. I missed it before, but result is the same unfortunately.
![screen-28-01-2022_07-36](https://user-images.githubusercontent.com/18647764/151488707-e9b5ca0b-bbee-4ccc-91c7-e6f818472b88.png)
